log4j-core/src/main/java/org/apache/logging/log4j/core/LoggerContext.java

```diff
@@ -436,13 +435,7 @@ public boolean stop(final long timeout, final TimeUnit timeUnit) {
             }
 
             this.setStopping();
-            String name = getName();
-            try {
-                Server.unregisterLoggerContext(name); // LOG4J2-406, LOG4J2-500
-            } catch (final LinkageError | Exception e) {
-                // LOG4J2-1506 Hello Android, GAE
-                LOGGER.error("Unable to unregister MBeans", e);
-            }
+
```

Comment:
Why did we remove this and another below Android-specific guards?

Reply:
I am not sure I understand: I removed the `Server` class and method call, so the `try ... catch` becomes empty.

Reply:
Oh! :facepalm: Got it.